extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -740,6 +747,12 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		return undefined;
 	}
 
+	private async getInstalledPythonVersion(pythonExecutable: string): Promise<string> {
+		let cmd = `"${pythonExecutable}" -c "import platform;print(platform.python_version())"`;
```

Comment:
Can't just use `python -v`?

Reply:
Using the --version option prints a long string that we have to parse the version number out of. Using python_version() gives us just the version number. For example, when using an Anaconda version of python, "python --version" prints the following: "Python 3.6.12 :: Anaconda, Inc."